cachito/web/api_v1.py

```diff
@@ -473,3 +486,46 @@ def add_request_config_files(request_id):
 
     db.session.commit()
     return "", 204
+
+
+def generate_response_(log_file_path):
+    """
+    Generate response by streaming the content.
+
+    :param str log_file_path: file path to read content from
+    :return: streamed content for the given file
+    :rtype: Generator[str, None, None]
+    """
+    with open(log_file_path) as f:
+        for line in f:
```

Comment:
Can you stream more than just a single line at a time? Like 1024 bytes at a time seems reasonable.